### ISSUE TITLE
Crash on Startup

### ISSUE THREAD

**Issue by asantoni** [solvespace/solvespace#501]
### System information

SolveSpace version: 3.0 git revision 5d78f993cea3749b6644e0a2f9ddf89c9754e377

Operating system: Windows 10

### Expected behavior 

No crash...

### Actual behavior

![image](https://user-images.githubusercontent.com/569913/69391300-6363b180-0ca0-11ea-904f-e5cd99f8c4cb.png)

### Additional information

This was working yesterday for me, but mysteriously stopped working today. Sorry for the image paste of the backtrace, but I hope it's still useful. I'm not sure I have the resolve to track down this crash further myself...

**Comment by asantoni** [comment]
Sorry, that backtrace was cut off... Here's the full backtrace:

![image](https://user-images.githubusercontent.com/569913/69391391-ade52e00-0ca0-11ea-9c1b-8989fb3a8fba.png)

**Comment by whitequark** [comment]
Does this patch help?
```diff
diff --git a/src/render/gl3shader.cpp b/src/render/gl3shader.cpp
index 8ad45c7a..542eb97f 100644
--- a/src/render/gl3shader.cpp
+++ b/src/render/gl3shader.cpp
@@ -390,7 +390,7 @@ GLuint Generate(const std::vector<double> &pattern) {
     glGetIntegerv(GL_MAX_TEXTURE_SIZE, &size);
     RgbaColor *textureData = new RgbaColor[size];
 
-    int mipCount = (int)log2(size) + 1;
+    int mipCount = (int)log2(size);
     for(int mip = 0; mip < mipCount; mip++) {
         int dashI = 0;
         double dashT = 0.0;
```

**Comment by asantoni** [comment]
Yep, that fixed it! Thanks for the quick fix!

**Comment by asantoni** [comment]
Whoops, I should have left closing this ticket up to you.

**Comment by whitequark** [comment]
> This was working yesterday for me, but mysteriously stopped working today.

Did you perhaps rebuild the application with a debug C++ runtime?

**Comment by whitequark** [comment]
Fixed in 91ac6665.
